applications/nrf_desktop/doc/selector.rst

```diff
@@ -28,7 +28,8 @@ For this reason, you should set :kconfig:`CONFIG_GPIO` option.
 
 Set :kconfig:`CONFIG_DESKTOP_SELECTOR_HW_ENABLE` option to enable the module.
 The configuration for this module is an array of :c:struct:`selector_config` pointers.
-The array is written in the :file:`selector_hw_def.h` file located in the board-specific directory in the application configuration directory.
+The array is written in the configuration file specified by the :kconfig:`CONFIG_DESKTOP_SELECTOR_HW_DEF_PATH` option.
+The configuration file should be placed in the board-specific directory in the application configuration directory.
```

Comment:
I suggest you mention this option in a list of Kconfig options.
If this option is mandatory, it would be good to rewrite the whole part in lines 26-41 into steps.